static/go101/css/v7.css

```diff
@@ -0,0 +1,16 @@
+@media (min-width: 798px) {.container {max-width: 798px;}}
+
+table.list td {vertical-align:top; padding-right:0.5em; padding-bottom:0.2em;}
+.affix {position: fixed; top: 30px;}
+pre code {background-color: inherit; border-style: none;}
+.alert code {background-color: inherit; border-style: solid; border-width: 1px; padding-top:3px; margin-left: 1px; margin-right: 1px;}
+div code, p code {border-width: 3px; padding-top:1px; padding-bottom: 1px;}
+pre.output {background-color:#222; color:#ffff00;}
+.anchor {display:block; height:10px; margin-top:-16px; visibility:hidden;}
+.nav-bar-with-borders {border-top: 1px solid #6886c1; border-bottom: 1px solid #6886c1; margin: 16px 0px; padding: 3px 6px;}
+.nav-item-active {text-align: center; padding: 3px 0px; background-color: #d9edf7;}
+.nav-item-inactive {text-align: center; padding: 3px 0px;}
+.social-button {vertical-align:top; margin-top: 2px; margin-bottom: 2px;}
+li {margin: 10px 5px 10px 0}
+p, li {text-align: justify !important;}
+/* TODO Define a NOTE box for notes in the articles. */
```

Comment:
Aha, you also agree NOTE needs a special style.

Texts are justified and margin Not bad. Let's try it and see the effect.

I fell the margin (10px) for list items is too large. Now it is 0px.
Let's make a trade-off, how about 3px? :)

Add a `.note` style here?
```
.note {}
```